platform/darwin/src/MGLGeoJSONOptions.h

```diff
@@ -0,0 +1,13 @@
+#import <Foundation/Foundation.h>
+
+@interface MGLGeoJSONOptions : NSObject
+
+@property (nonatomic, assign) NSInteger maximumZoom;
```

Comment:
Thanks for spelling out “maximum”. For consistency with `-[MGLMapView maximumZoomLevel]`, I think this should be called `maximumZoomLevel` too.